lib/src/kbasesearchengine/common/JsonTokenUtil.java

```diff
@@ -64,7 +65,7 @@ public static ObjectJsonPath trimPath(ObjectJsonPath jsonPath) {
      * we read field token before processing value block.
      */
     public static void writeTokensFromCurrent(final JsonParser jts, final JsonToken current,
-            final JsonGenerator jgen) throws IOException {
+            final JsonGenerator jgen) throws ObjectParseException, IOException {
```

Comment:
Is IOExeption thrown from this method anymore? If not, remove. Same question for the other methods that throw IOException.

Reply:
Yes it is being thrown by JsonGenerator.writeStartArray()